RecoEgamma/EgammaHLTProducers/plugins/EgammaHLTR9IDProducer.cc

```diff
@@ -82,7 +82,11 @@ void EgammaHLTR9IDProducer::produce(edm::StreamID sid, edm::Event& iEvent, const
   reco::RecoEcalCandidateIsolationMap r95x5Map(recoecalcandHandle);
   for (unsigned int iRecoEcalCand = 0; iRecoEcalCand < recoecalcandHandle->size(); iRecoEcalCand++) {
     reco::RecoEcalCandidateRef recoecalcandref(recoecalcandHandle, iRecoEcalCand);  //-recoecalcandHandle->begin());
-
+    if (recoecalcandref->superCluster()->seed()->seed().det() != DetId::Ecal) {     //HGCAL, skip for now
+      r9Map.insert(recoecalcandref, 0);
+      r95x5Map.insert(recoecalcandref, 0);
```

Comment:
actually the default value for HGCAL should be 1 (not 0). Because R9 cut in phase2 HLT paths will be of the form `R9>some_value_around_0.9`. So if it's set to 1 in HGCAL then it will always pass, as intended. I will make this change soon.